app/src/main/java/com/github/se/project/ui/components/WritableDropdown.kt

```diff
@@ -0,0 +1,59 @@
+package com.github.se.project.ui.components
+
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.material3.DropdownMenu
+import androidx.compose.material3.DropdownMenuItem
+import androidx.compose.material3.OutlinedTextField
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.window.PopupProperties
+import androidx.compose.ui.zIndex
+
+@Composable
+fun WritableDropdown(
```

Comment:
this is a good composable idea for modularisation and we should implement this in other screens :)